install/windows/docker-ee.md

```diff
@@ -1,23 +1,23 @@
 ---
-description: How to install Docker EE for Windows Server
-keywords: Windows, Windows Server, install, download, ucp, Docker EE
-title: Install Docker Enterprise Edition for Windows Server
+description: How to install Docker Engine - Enterprise for Windows Server
+keywords: Windows, Windows Server, install, download, ucp, Docker Engine - Enterprise
+title: Install Docker Engine - Enterprise on Windows Servers
 redirect_from:
 - /docker-ee-for-windows/install/
 - /engine/installation/windows/docker-ee/
 ---
 
 {% capture filename %}{{ page.win_latest_build }}.zip{% endcapture %} {% capture download_url %}https://download.docker.com/components/engine/windows-server/{{ site.docker_ee_version }}/{{ filename }}{% endcapture %}
 
-Docker Enterprise Edition for Windows Server (*Docker EE*) enables native Docker containers on Windows Server. Windows Server 2016 and later versions are supported. The Docker EE installation package includes everything you need to run Docker on Windows Server.  This topic describes pre-install considerations, and how to download and install Docker EE.
+Docker Engine - Enterprise enables native Docker containers on Windows Server. Windows Server 2016 and later versions are supported. The Docker Engine - Enterprise installation package includes everything you need to run Docker on Windows Server.  This topic describes pre-install considerations, and how to download and install Docker Engine - Enterprise.
 
 > Release notes
 >
 > [Release notes for all versions](/release-notes/)
 
-## Install Docker EE
+## Install Docker Engine - Enterprise
 
-Docker EE for Windows requires Windows Server 2016 or later. See
+Docker Engine - Enterprise requires Windows Server 2016 or later. See
 [What to know before you install](#what-to-know-before-you-install) for a
 full list of prerequisites.
 
```

Comment:
This link is not helpful at all in the install process - it needs to be a system requirements page. As a customer I would expect that link to actually tell me about pre-install configuration and system requirements.
This one gives no useable information, it's a link to other links and it's in a confusing location in the doc near Next steps  https://docs.docker.com/install/windows/docker-ee/#what-to-know-before-you-install

Reply:
I get what you're saying, but the only other link I can find is the Docker Desktop for Windows information. Do you know who would have the Windows system requirements? I would put the directly in this document for now.

Reply:
@jasonbivins is going to provide me with the Windows server system requirements information, and after it's committed, I'll push this off for peer review.

Reply:
Here's the sysreqs page from MSFT: 
https://docs.microsoft.com/en-us/virtualization/windowscontainers/deploy-containers/system-requirements

Reply:
@jasonbivins thanks for the information, but I need to have the development team provide me information on what our system requirements are and put that directly in the documentation. CC: @carlfischer1 and @ddebroy

Reply:
@ddebroy was able to get me the information.